proto/p4/p4runtime.proto

```diff
@@ -356,6 +357,30 @@ message CloneSessionEntry {
   uint64 packet_length_bytes = 4;
 }
 
+//------------------------------------------------------------------------------
+// For writing and reading matches in a parser value set. A state transition
+// on an empty value set will never be taken. The number of matches must be at
+// most the size of the value set as specified by the @size annotation in P4
+// program. The priority of matches (high to low) is the order in which they
+// have been defined in the repeated field.
```

Comment:
I believe that a single value set, no matter how many entries it contains, there is no difference in the parsing behavior regardless of which entry a packet matches.  i.e. it can be considered as an unordered set, with no significance to which entry matches, only whether any of them do, or none of them do.

Reply:
Correct, that's my bad I misled @samar-abdi :/